test/src/e2e.dynval/dv.n+1.test.ts

```diff
@@ -0,0 +1,249 @@
+            const nominateTxHash = checkingNode.sdk.rpc.chain.sendSignedTransaction(
+                nominateTx
+            );
+            const delegateTx = stake
+                .createDelegateCCSTransaction(
+                    checkingNode.sdk,
+                    betty.platformAddress,
+                    5_000
+                )
+                .sign({
+                    secret: faucetSecret,
+                    seq: faucetSeq + 1,
+                    fee: 10
+                });
+            const delegateTxHash = checkingNode.sdk.rpc.chain.sendSignedTransaction(
+                delegateTx
+            );
+            await checkingNode.waitForTx([nominateTxHash, delegateTxHash]);
+            await bettyInsertionCheck(checkingNode);
+        });
+    });
+
+    describe("Increase one candidate's deposit which is less than the minimum deposit", async function() {
+        const nodes = withNodes(this, {
+            promiseExpect,
+            validators: otherDynValidators
+                .map((signer, index) => ({
+                    signer,
+                    delegation: 5_000,
+                    deposit: 10_000_000 - index // tie-breaker
+                }))
+                .concat([
+                    {
+                        signer: betty,
+                        delegation: 5_000,
+                        deposit: 9999
+                    }
+                ]),
+            overrideParams: {
+                termSeconds,
+                maxNumOfValidators: 8
+            }
+        });
+
+        it("betty shoule be included in validators", async function() {
+            const checkingNode = nodes[0];
+            await beforeInsertionCheck(checkingNode);
+            const nominateTx = stake
+                .createSelfNominateTransaction(checkingNode.sdk, 10_000, "")
+                .sign({
+                    secret: betty.privateKey,
+                    seq: await checkingNode.sdk.rpc.chain.getSeq(
+                        betty.platformAddress
+                    ),
+                    fee: 10
+                });
+            const nominateTxHash = checkingNode.sdk.rpc.chain.sendSignedTransaction(
+                nominateTx
+            );
+            await checkingNode.waitForTx(nominateTxHash);
+            await bettyInsertionCheck(checkingNode);
+        });
+    });
+
+    describe("Delegate more stake to which was less than the minimum delegation", async function() {
+        const nodes = withNodes(this, {
+            promiseExpect,
+            validators: otherDynValidators
+                .map((signer, index) => ({
+                    signer,
+                    delegation: 5_000,
+                    deposit: 10_000_000 - index // tie-breaker
+                }))
+                .concat([
+                    {
+                        signer: betty,
+                        delegation: 999,
+                        deposit: 10_000_000 - otherDynValidators.length
```

Comment:
It doesn't have to subtract `otherDynValidators.length`. The tie is between nodes that have the same amount of delegation.

Reply:
Ah, Ok. I changed it.